crates/usvg/src/parser/paint_server.rs

```diff
@@ -331,7 +331,7 @@ fn convert_stops(grad: SvgNode) -> Vec<Stop> {
     if stops.len() >= 3 {
         let mut i = 0;
         while i < stops.len() - 2 {
-            let offset1 = stops[i + 0].offset.get();
+            let offset1 = stops[i].offset.get();
```

Comment:
Again... I can see the rationale, but I think the previous version was probably better.